packages/mongodb-chatbot-eval/src/commands/evaluate.ts

```diff
@@ -1,22 +1,38 @@
 import { CommandModule } from "yargs";
 import { LoadConfigArgs, withConfig, withConfigOptions } from "../withConfig";
 import { EvalConfig } from "../EvalConfig";
+import { generateEvalsAndMetadata } from "../evaluate/generateEvalsAndMetadata";
+import { ObjectId } from "mongodb-rag-core";
 
-const commandModule: CommandModule<unknown, LoadConfigArgs> = {
+interface EvaluateCommandArgs {
+  name: string;
+  generatedDataRunId: string;
+}
+
+const commandModule: CommandModule<
+  unknown,
+  LoadConfigArgs & EvaluateCommandArgs
+> = {
   command: "evaluate",
   builder(args) {
     return withConfigOptions(args)
-      .string("name")
-      .option("generatedDataQuery", {
+      .option("name", {
         type: "string",
-        description: "Query to filter generated data.",
+        demandOption: true,
+        description: "Name of the evaluation.",
       })
-      .demandOption("generatedDataQuery")
-      .demandOption("name");
+      .option("generatedDataRunId", {
+        type: "string",
+        description:
+          "RunId for a 'generate' command that you want to create evaluations for. If not provided, uses the `defaultGeneratedDataQuery` from the config. Throws an error if no filter provided here or there's no default.",
+        demandOption: true,
+      });
   },
   async handler(args) {
     return withConfig(evaluateCommand, {
       ...args,
+      name: args.name,
+      generatedDataRunId: args.generatedDataRunId,
```

Comment:
I think we don't need `name` and `generatedDataRunId` here because they're just repeating fields that are already included in the `args` spread

Reply:
you right, i had it the other way when i was doing the casting. fixed